plugins/recruit-resources/src/components/CreateCandidate.svelte

```diff
@@ -116,11 +115,19 @@
     return value === undefined || value === ''
   }
 
-  function addChannel (channels: AttachedData<Channel>[], type: Ref<ChannelProvider>, value?: string): void {
+  function addChannel (channels: Channel[], type: Ref<ChannelProvider>, value?: string): void {
     if (value !== undefined) {
       const provider = channels.find((e) => e.provider === type)
       if (provider === undefined) {
         channels.push({
+          _id: generateId(),
```

Comment:
Ok, but why not use AttachedData<Channel>? If all this fields will not be used on creation or update?